packages/merge-styles/src/styleToClassName.ts

```diff
@@ -177,6 +177,18 @@ function getKeyForRules(options: IStyleOptions, rules: IRuleSet): string | undef
   return hasProps ? serialized.join('') : undefined;
 }
 
+function repeatString(target: string, count: number): string {
+  if (count <= 0) {
```

Comment:
`createArray(count), () => target).join('')`

Reply:
maybe the 2nd is more efficient anyway. This will be a hotspot called for every selector.

Reply:
recursion is actually the most performant for this use case. also, we should never expect a large `count` here. let me know if you disagree